pkgs/development/libraries/ctranslate2/default.nix

```diff
@@ -49,6 +53,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = lib.optionals withMkl [
     mkl
+  ] ++ lib.optionals withCUDA [
+    cudatoolkit
```

Comment:
Could you try if this works with `cudaPackages.*` instead? It's `cuda_nvcc` in `nativeBuildInputs` and `cuda_cudart`&c in `buildInputs`. The run-file based `cudatoolkit` is being phased out

Reply:
Thanks, updated.